birthdays/static/styles.css

```diff
@@ -0,0 +1,92 @@
+body {
+    margin: 0;
+    font-size: 1rem;
+    font-weight: 400;
+    line-height: 1.5;
+    color: #212529;
+    text-align: left;
+    background-color: #fff;
```

Comment:
For all of these CSS properties in this file (and the other file for the other lab), let's alphabetize the properties for consistency with lecture code (e.g. `background-color` before `color`).